### ISSUE TITLE
Toolboxes to move to mature support

### ISSUE THREAD

**Issue by mfunk** [Esri/solutions-geoprocessing-toolbox#461]
List of toolboxes that will be moved to mature support as their templates are moved to mature support as well.
* [x] Point Of Origin Tools
    * Point Of Origin Tools.tbx
    * Point Of Origin Tools_10.4.tbx
    * Point Of Origin Tools_10.3.tbx
* [x] CADRG ECRG Tools
    * CADRG ECRG Tools.pyt
* [x] CIB Tools
    * CIB Tools.pyt
* [x] Elevation Tools
    * Elevation Tools.pyt
* [x] Import and Conversion Tools
    * Import and Conversion Tools.tbx
    * Import and Conversion Tools_10.4.tbx
* [x] Patrol Data Capture Tools
    * Patrol Data Capture Tools.tbx
    * Patrol Data Capture Tools_10.4.tbx
* [x] Range Card Tools
    * Range Card Tools.tbx
    * Range Card Tools_10.4.tbx

Toolboxes with no templates that are not used and will be moved to mature support
* [x] Network Data Preparation Tools
    * Network Data Preparation Tools.tbx
    * Network Data Preparation Tools_10.4.tbx
* [x] Publishable Task Tools
    * Publishable Task Tools.tbx
    * Publishable Task Tools_10.4.tbx

**Comment by mfunk** [comment]
@nfeuerstein So the above lists are the stuff that we have identified to move to mature support. We've noted the first list as part of our template deprecation for Nov 2016. The other two are not part of a template but aren't part of any process that we have used in over 2 years. These are a housekeeping deprecation.

Later on we will need to remove the toolboxes and all of the associated files as part of a separate issue. I'll log that one next and set for a future milestone.
